spec/rails_helper.rb

```diff
@@ -34,6 +34,7 @@
   config.before(:suite) do
     DatabaseCleaner.strategy = :transaction
     DatabaseCleaner.clean_with(:truncation)
+    InterventionType.seed_somerville_intervention_types
```

Comment:
The spec was failing on the controller code calling `intervention.name`, it trying to delegate that to `intervention_type`, but there being no records in that table.